Alpha team/style.css

```diff
@@ -0,0 +1,938 @@
+
+}
+
+/* Hewr End */
+
+/* Randa Start */
+.experience-content {
+  padding: 15px;
+}
+
+.experience-content>h2 {
+  font-size: 18px;
+  line-height: 30px;
+  font-weight: 700;
+  color: #3f3849;
+}
+
+.experience-addresss>h6 {
+  font-size: 14px;
+  line-height: 30px;
+  font-weight: 700;
+  color: #aca3ba;
+}
+
+.experience-addresss>span {
+  font-size: 38px;
+  font-weight: bold;
+  color: #3F3849;
+}
+
+.progress-div {
+  position: relative;
+  ;
+}
+
+.progress-text {
+  position: absolute;
+  color: black;
+  bottom: 1.7rem;
+  right: 0;
+  z-index: 3;
+  font-size: 20px;
+}
+
+.progress {
+  height: 7px;
+}
+
+.progress-bar {
+  background: #7C1EFD;
+  width: 0;
+}
+
+.progress-item1 {
+  transition: all;
+  animation: progress-animate1 .6s ease
+}
+
+@keyframes progress-animate1 {
+  0% {
+    width: 0%;
+  }
+
+  50% {
+    width: 15%;
+  }
+
+  100% {
+    width: 30%;
+  }
+}
+
+.progress-item2 {
+  transition: all;
+  animation: progress-animate2 .6s ease;
+}
+
+@keyframes progress-animate2 {
+  0% {
+    width: 0%;
+  }
+
+  25% {
+    width: 15%;
+  }
+
+  50% {
+    width: 30%;
+  }
+
+  100% {
+    width: 80%;
+  }
+}
+
+.progress-item3 {
+  transition: all;
+  animation: progress-animate3 .6s ease-in-out;
+}
+
+@keyframes progress-animate3 {
+  0% {
+    width: 0%;
+  }
+
+  25% {
+    width: 15%;
+  }
+
+  50% {
+    width: 30%;
+  }
+
+  100% {
+    width: 60%;
+  }
+}
+
+/* experience img*/
+
+
+/*end Best Experience (by randa)*/
+
+/*  Start what our customers say (done by randa) */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+
+
+.slide-addres {
+  text-align: center;
+  color: #3F3849;
+
+
+}
+
+.silder-area {
+  background: #F4F4F4;
+  padding: 50px 0;
+
+  position: relative;
+}
+
+.img-area {
+  margin: 0 auto;
+  width: 100px;
+  height: 110px;
+  overflow: hidden;
+  text-align: center;
+}
+
+.img-area img {
+  border-radius: 50%;
+  box-shadow: 0 5px 20px rgba(0, 0, 0, .05), 0 5px 20px rgba(0, 0, 0, .05);
+}
+
+.carousel-caption {
+  position: static;
+  padding-top: 0;
+  padding-bottom: 15px;
+}
+
+.carousel-caption>p {
+  color: #8D8698;
+  margin: auto;
+  width: 60%;
+  font-size: 18px;
+}
+
+.carousel-caption>span {
+  color: #8D8698;
+  font-size: 22px;
+}
+
+.carousel-caption>h3 {
+  font-size: 26px;
+  margin-bottom: 15px;
+  margin-top: 25px;
+  color: #7C1EFD;
+}
+
+/* Ahmed start */
+.buy-btn {
+
+  background-color: #7D1EFF;
+  /* Green */
+  padding: 10px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.buy-btn:hover {
+  background: #AA6DFF;
+}
+
+.header-properties {
+  background-color: black;
+  opacity: .9;
+  color: gray;
+  height: 100vh;
+  font-size: 16px;
+}
+
+.dim {
+  max-width: 550px;
+}
+
+.font-size-header {
+  font-size: 61px;
+}
+
+.font-size-header div {
+  color: blueviolet;
+  display: inline;
+  padding: 0.25rem;
+}
+
+.nav-proper {
+  opacity: 0.9;
+  color: inherit;
+  z-index: 22;
+  padding: 4% 3%;
```

Comment:
avoid using % with paddings lots of times they aren't really effective, try to use terms or pixels